### PR TITLE
CVE-2018-10899

### PR DESCRIPTION
**Pull request by pedrohc**

### CODE REVIEW

**Comment by iamleot** [comment]
Hello Pedro,
according the reference that seems fixed in 1.6.1.
Can you adjust the description accordingly from:

 `A flaw was found in Jolokia versions from 1.2. up to and including 1.6.0.'

to:

 `A flaw was found in Jolokia versions from 1.2 before 1.6.1.'

Can you please also add:

 <https://jolokia.org/changes-report.html#a1.6.1>

to the reference?

File: 2018/10xxx/CVE-2018-10899.json [hunk 1]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2018/10xxx/CVE-2018-10899.json]: Invalid JSON file

**Comment by pedrohc** [comment]
Sorry, not sure whats wrong here, can you help?

**Comment by cve-team** [comment]
I believe you're just missing a comma after the "name" value and it should validate:

                        {
				"url": "https://jolokia.org/changes-report.html#a1.6.1",
				"name": "https://jolokia.org/changes-report.html#a1.6.1"
				"refsource": "CONFIRM"
			}

**Comment by cve-team** [comment]
Thanks, this ID has been populated and will be published on cve.mitre.org within 1-2 hours.

**Pull request merged**